README.md

```diff
@@ -312,6 +317,50 @@ export default withStyles(({ color, unit }) => ({
 
 `className` and `style` props must not be used on the same elements as `css()`.
 
+## Examples
+### With React Router's `Link`
+[React Router][react-router]'s [`<Link/>`][react-router-link] and [`<IndexLink/>`][react-router-index-link] components accept `activeClassName='...'` and `activeStyle={{...}}` as props. As previously stated, `css(...styles)` must spread to JSX. In order to mimic `activeClassName` you can use React Router's [`withRouter()`][react-router-with-router] Higher Order Component on top of `withStyles()` like so:
+
+```jsx
+import React from 'react';
+import { withRouter, Link } from 'react-router';
+import { css, withStyles } from '../withStyles';
+
+function Nav({ router, styles }) {
+  return (
+    <div {...css(styles.container)}>
+      <Link
+        to="/"
+        {...css(styles.link, router.isActive('/', true) && styles.link_bold)}
```

Comment:
Also, does this actually work? In the paragraph above, you mention that the Link API is `activeClassName` and `activeStyle`.

I'm not very familiar with react router, so maybe the withRouter HOC is doing something special here?